components/script/dom/xrrenderstate.rs

```diff
@@ -47,6 +46,25 @@ impl XRRenderState {
             XRRenderStateBinding::Wrap,
         )
     }
+
+    pub fn copy(&self) -> DomRoot<Self> {
```

Comment:
Bikeshed: this is cloning rather than copying?

Reply:
Yes but I didn't want to call it clone. Maybe deep_copy? Or clone_object.

Reply:
`shallow_clone()`?

Reply:
It's a deep clone though. I called it clone_object.

Reply:
Well, not 100% deep since it doesn't clone the JS side.